cdk/lib/frontend.ts

```diff
@@ -18,5 +34,384 @@ export class Frontend extends GuStack {
+          "dynamodb:DescribeTable",
+        ],
+        resources: [
+          membershipSubPromotionsTable,
+          "arn:aws:dynamodb:*:*:table/MembershipSub-Promotions-UAT",
+        ],
+      }),
+      new GuAllowPolicy(this, "DynamoRedemptions", {
+        actions: ["dynamodb:GetItem"],
+        resources: [
+          redemptionCodesTable,
+          "arn:aws:dynamodb:*:*:table/redemption-codes-UAT",
+        ],
+      }),
+      new GuAllowPolicy(this, "StripeSetupIntentLambda", {
+        actions: ["lambda:InvokeFunction"],
+        resources: [
+          `arn:aws:lambda:eu-west-1:${this.account}:function:stripe-intent-${this.stage}`,
+        ],
+      }),
+    ];
+
+    const ec2App = new GuEc2App(this, {
+      applicationPort: 9000,
+      app: "frontend",
+      access: { scope: AccessScope.PUBLIC },
+      certificateProps: {
+        [Stage.CODE]: {
+          domainName:
+            "support.code.theguardian.com.origin.membership.guardianapis.com",
+          hostedZoneId: "Z1E4V12LQGXFEC",
+        },
+        [Stage.PROD]: {
+          domainName:
+            "support.theguardian.com.origin.membership.guardianapis.com",
+          hostedZoneId: "Z1E4V12LQGXFEC",
+        },
+      },
+      monitoringConfiguration: {
+        noMonitoring: true,
+      },
+      userData,
+      roleConfiguration: {
+        additionalPolicies: policies,
+      },
+      scaling: {
+        PROD: {
+          minimumInstances: 3,
+          maximumInstances: 6,
+        },
+        CODE: {
+          minimumInstances: 1,
+          maximumInstances: 2,
+        },
+      },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.SMALL),
+    });
+
+    // TODO: remove this tag after the migration
+    const ec2AppAsg = ec2App.autoScalingGroup;
+    Tags.of(ec2AppAsg).add("gu:riffraff:new-asg", "true");
+
+    new GuAlarm(this, "NoHealthyInstancesAlarm", {
+      app,
+      alarmName: `URGENT 9-5 - ${this.stage} no healthy instances for support-frontend`,
+      alarmDescription:
+        "Impact - Cannot sell any subscriptions or contributions products. Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit",
+      actionsEnabled: this.withStageDependentValue({
+        app,
+        variableName: "NoHealthyInstancesAlarmEnabled",
+        stageValues: {
+          [Stage.CODE]: false,
+          [Stage.PROD]: true,
+        },
+      }),
+      threshold: 0.5,
+      evaluationPeriods: 2,
+      comparisonOperator: ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "HealthyHostCount",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+          TargetGroup: ec2App.targetGroup.targetGroupFullName,
+        },
+        statistic: "Average",
+        period: Duration.seconds(60),
+      }),
+      snsTopicName: "reader-revenue-dev",
```

Comment:
Would be good to get this from parameter store (see also https://github.com/guardian/cdk/pull/884).

Reply:
Yeah makes sense - would this be a follow up to do after that cdk PR has been merged?

Reply:
What is the advantage of getting it from parameter store? Wouldn't it just add another level of indirection making it harder to maintain and debug?